owncloudApp/src/main/java/com/owncloud/android/features/ReleaseNotesList.kt

```diff
@@ -0,0 +1,33 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author David Crespo Ríos
+ * Copyright (C) 2022 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.features
+
+import com.owncloud.android.datamodel.ReleaseNote
+
+class ReleaseNotesList {
+
+    companion object {
+        private val releaseNotesList = emptyList<ReleaseNote>()
+
+        fun getReleaseNotes(): List<ReleaseNote> {
+            return releaseNotesList
+        }
+    }
+}
```

Comment:
Do you mean leave it as a val in the viewModel? I can´t use it there because I need the list before initializing the viewModel (to know if the release notes should show), so I wouldn´t have the releaseNotesList in the method `shouldShow`

Reply:
Now in the companion object of the viewModel. Done